README.md

```diff
@@ -1,4 +1,7 @@
-#Concat
+#concat
+
+
+[![Build Status](https://travis-ci.org/puppetlabs/puppetlabs-concat.png?branch=master)](https://travis-ci.org/puppetlabs/puppetlabs-concat)
```

Comment:
We were getting rid of these so we could disable an extra layer of travis-ci testing to ease the backup, so can you remove the build status link?